glfs.c

```diff
@@ -563,20 +563,24 @@ static int tcmu_glfs_open(struct tcmu_device *dev, bool reopen)
 		if (round_down(st.st_size, block_size) == dev_size)
 			goto out;
 
-		if (!reopen) {
-			ret = -EINVAL;
-			goto close;
-		}
+		if (!reopen)
+			tcmu_dev_warn(dev, "Device size mismatch during initialization.\n");
```

Comment:
may start with the small case just like in other places:
"Device size..." -> "device size..."

Reply:
Checked all the other in tcmu-runner and glfs, they are all start with upper case. And only some start with function name will the be small case.